libs/client/features/src/accounts-manager/stock/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './AddStock'
```

Comment:
Ahh, good old index files. I'm not opposed to this, but find them to be a waste of time. VSCode can auto-import files, so why add this at all?

Reply:
Works for me, just pointing out it's a waste of time when working with modern JS / TS (aka with ES Modules). Just use named exports when possible, and let VSCode deal with imports for you.